_data/tools.yml

```diff
@@ -471,6 +471,30 @@
       url: https://trino.io/episodes/30.html
     - urltext: Introduction to dbt and Trino from Trino Community Broadcast 21
       url: https://trino.io/episodes/21.html
+- name: DbVisualizer
+  anchor: dbvisualizer
+  category: client
+  logo: /assets/images/logos/dbvisualizer.png
+  logosmall: /assets/images/logos/dbvisualizer-small.png
+  description: |
+    DbVisualizer has a long history of being a top-performing Trino editor.
+
+    Use DbVisualizer to easily connect to and manage your Trino databases. 
+    With extended support for Trino specific object types you can enjoy the full suite 
+    of smart features that DbVisualizer has to offer.
+  links:
+    - urltext: Mastering the Trino Connection with DbVisualizer
+      url: https://www.dbvis.com/thetable/mastering-the-trino-connection-unleash-the-power-of-dbvisualizer/
+    - urltext: DbVisualizer Website
+      url: https://www.dbvis.com/
+    - urltext: DbVisualizer Docs
```

Comment:
I removed the links to features och object type support since those can be reached from the overview. Docs however is not a visible link on www.dbvis.com (it really should be) so I left it in the link list. Also kept the blog post in the link list.

Reply:
ok